### ISSUE TITLE
[JetEngine] Extend admin columns display options for CCTs

### ISSUE THREAD

**Issue by oceandiveloper** [CrocoBlock/suggestions#6806]
The backend layout of CCT items in the admin dashboard is currently very limited. For CPTs I can simply use Admin Columns Pro to display the table view like I like it.

For CCTs there is nothing available to:
1. Display related items (including inline editing support)
2. change the width of an admin column (I currently to that with custom Admin CSS)
3. change in which way the values are displayed (e.g. user first name instead of user name, proper image display for media fields, adjusting date format for datetime fields)
4. allow for horizontal scrolling when there are to many columns and the display limit is reached to comfortably read the CCT item values.
5. reorder the admin columns

**Comment by Allespat** [comment]
Would be an important addition...

**Comment by mojtabaiam** [comment]
This option is very needed.
+1

**Comment by ashentertain** [comment]
Yes, please also add the option to show the field label instead of value in the admin column.
